### DISCUSSION TITLE
Using full width for newly added/deleted files in split view

### DISCUSSION THREAD

**Discussion by f-g-p** [community/community#36239] [Pull Requests]
Hi,
I only use the split view to review PRs, and in this view newly added files only use half of the available space on screen, which is kind of a pity (poor formatting like text wrapping). 

Is there a good reason to keep it like this ? 
couldn't it be made full-width ?

Thanks a lot

**Comment by jfullstackdev** [comment]
what do you mean by that ? 

you can change the view

![image](https://user-images.githubusercontent.com/47092464/195994327-b39c11f3-3fea-4edb-9af1-3dfb61d18571.png)

**Comment by uniquecoderpriya** [comment]
ya, offcourse you can change this view .

**Comment by f-g-p** [comment]
Hi, 
Sorry If I wasn't clear. 
I know I can change the view. I am speaking about the Split view only, and for newly added files within the PR only. 
<img width="1408" alt="image" src="https://user-images.githubusercontent.com/8312942/197549878-a5c4ad66-edf6-432c-93c4-f185f29b20b9.png">

basically I would like to use split view for edited files, and full width for new and removed files

**Comment by jfullstackdev** [comment]
well, that would depend on the PR too, because the developer will decide what will be the content included in a PR, it's not just one big PR with lots of commits doing

usually for me, PRs will be always adding a new feature or even partial feature because that way you can focus on what is the intention of the PR,

so in your case, if that is the kind of PR, viewing in Split view can be easily achieve and say, another PR, just only for addition or deletion of files

**Reply by f-g-p** [reply]
humm, 
I'm sorry I don't see how this can be a good solution. If someone needs to add 2 files and modify 3 others for its change, I want to review them together (same context). Plus we have some Github actions to compile, tests ect. that would completely be broken by this approach
